devel/external/.gitignore

```diff
@@ -1 +1,4 @@
 temp*
+/test_photon_vs_fft/
+/test_reconvolution/
+/test_sersic_highn/
```

Comment:
Why are you ignoring these?  We want to keep these in the repo.  Please revert this.